### PR TITLE
Refactor project to workspace

### PR DESCRIPTION
**Pull request by Vid201**
- Refactor project to workspace
- Add licenses to the project

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,73 +1,20 @@
...
+    "crates/contracts",
+    "crates/grpc",
+    "crates/primitives",
+    "crates/rpc",
+    "crates/uopool",
+    "tests",
+]
+default-members = ["bin/bundler"]
+
+[profile.debug-fast]
```
Comment (zsluedem) [inline Cargo.toml:14]: I am curious what this means. I could not google any good results on it. Maybe you could provide some links or description?

Comment (zsluedem) [inline Cargo.toml:14]: I am curious what this means. I could not google any good results on it. Maybe you could provide some links or description?

**Review by zsluedem — APPROVED** [review]
This is great work!! I only have one question. It should be good to merge.

File: Cargo.toml (hunk 1 above)
Comment (Vid201) [reply Cargo.toml:14]: This enables us to to do: `cargo build --profile debug-fast` and have debug build for debugging with high optimizations (as in release).

Link (example): https://doc.rust-lang.org/cargo/reference/profiles.html#custom-profiles

I saw this in reth and it looked useful: https://github.com/paradigmxyz/reth/blob/main/Cargo.toml

**Pull request merged**